.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
```

Comment:
- [ ] Please make sure you install all the necessary linters suggested by Microverse. Please follow all the steps listed in [this file](https://github.com/microverseinc/linters-config/tree/master/html-css).